test/repos/bom.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -ex; rm -rf repo; mkdir repo; cd repo
+
+git init
+
+cat <<EOT >> bomtest.txt ﻿A,B,C,D,E

```

Comment:
When I run this I get 
```
+ rm -rf repo
+ mkdir repo
+ cd repo
+ git init
Initialized empty Git repository in /Users/jesseduffieldduffield/go/src/github.com/jesseduffield/lazygit/test/repos/repo/.git/
+ cat A,B,C,D,E
cat: A,B,C,D,E: No such file or directory
```
Perhaps <<EOT should come after bomtest.txt? Or maybe this is a platform thing (I'm on OSX)